samples/sampler/stories/default/ImageItem.js

```diff
@@ -5,9 +5,9 @@ import ri from '@enact/ui/resolution';
 import ImageItem, {ImageItemBase} from '@enact/agate/ImageItem';
 
 const src = {
-	'hd':  'http://via.placeholder.com/200x200/9037ab/ffffff&text=Image0',
-	'fhd': 'http://via.placeholder.com/300x300/9037ab/ffffff&text=Image0',
-	'uhd': 'http://via.placeholder.com/600x600/9037ab/ffffff&text=Image0'
+	'hd':  'http://via.placeholder.com/200x200/9037ab/ffffff/png?text=Image0',
+	'fhd': 'http://via.placeholder.com/300x300/9037ab/ffffff/png?text=Image0',
+	'uhd': 'http://via.placeholder.com/600x600/9037ab/ffffff/png?text=Image0'
```

Comment:
I don't think we should change this .
We should be able to see that which image is loaded depending on the resolution.

Reply:
Yes, you are right. I will undo this.